subprojects/testing-base/src/test/groovy/org/gradle/api/internal/tasks/testing/logging/DefaultTestLoggingContainerTest.groovy

```diff
@@ -48,7 +48,13 @@ class DefaultTestLoggingContainerTest extends Specification {
         def logging = container.get(LogLevel.WARN)
 
         expect:
-        hasUnchangedDefaults(logging)
+        logging.events == [TestLogEvent.FAILED] as Set
```

Comment:
:o: Nitpick. We probably could reduce some duplication with these assertions and others in this class.